storage/acl.go

```diff
@@ -174,12 +175,12 @@ func (a *ACLHandle) bucketDelete(ctx context.Context, entity ACLEntity) error {
 func (a *ACLHandle) objectList(ctx context.Context) ([]ACLRule, error) {
 	var acls *raw.ObjectAccessControls
 	var err error
-	err = runWithRetry(ctx, func() error {
+	err = run(ctx, func() error {
 		req := a.c.raw.ObjectAccessControls.List(a.bucket, a.object)
 		a.configureCall(ctx, req)
 		acls, err = req.Do()
 		return err
-	})
+	}, a.retry, true)
```

Comment:
You are missing a bunch of calls in this file (mostly stuff that is not retried currently)-- could you add `run()` wrappings for those as well?

Reply:
Whoops - that brings to mind, should/can we test to check that all calls use the retry wrapping? Perhaps using the same list of functions as the retry conformance tests?

Reply:
Hmm, this is a good question. I'm not sure of a great way to test this! Let's think about it though (and we should definitely do an audit after the work is complete).